config/locales/forms/en.yml

```diff
@@ -22,11 +22,8 @@ en:
         labels:
           password: New password
       show: Show password
-    reactivate_profile:
-      instructions: Provide the information below to reactivate your account.
-      submit: Reactivate profile
-      title: Reactivate profile
     personal_key:
+      alternative_html: Don't have your personal key? %{link}
```

Comment:
I think this doesn't need `_html` because if the link is just at the end, we can just put the link right after the text in the view right?